src/main/java/edu/cornell/kfs/concur/service/ConcurRequestExtractFileService.java

```diff
@@ -0,0 +1,10 @@
+package edu.cornell.kfs.concur.service;
+
+import org.kuali.kfs.sys.batch.InitiateDirectory;
+
+/**
+ * Handles processing (validation, loading, and reporting) of incoming Concur Request Extract files.
+ */
+public interface ConcurRequestExtractFileService extends InitiateDirectory {
+	
```

Comment:
What methods do we expect in this service? Is this service responsible for the creating the pdp feed from the incoming request extract?